frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/PipelineWorkspaceSuggestionIcon.tsx

```diff
@@ -8,13 +8,16 @@ import './PipelineWorkspaceSuggestionIcon.scss';
 const PipelineWorkspaceSuggestionIcon: React.FC = () => {
   const { t } = useTranslation();
 
+  const ariaLabel = t('pipelines-plugin~Open hint');
   const content = t(
     "pipelines-plugin~Resources aren't in beta, so it is recommended to use workspaces instead.",
   );
 
   return (
     <Popover aria-label={content} bodyContent={content}>
-      <InfoCircleIcon className="opp-pipeline-workspace-suggestion-icon" />
+      <Button isInline variant="link" aria-label={ariaLabel}>
```

Comment:
```suggestion
      <Button isInline variant={ButtonVariant.link} aria-label={ariaLabel}>
```

Reply:
Good point. Copy and pasted this from another place. :)

@vikram-raj Updated :+1:

Reply:
@jerolimov so you also fixed the other place, right? :)